### PR TITLE
Search in fullname, email and fix login on Volto frontend when already logged-in in Plone Classic

### PR DESCRIPTION
**Pull request by avoinea**

### CODE REVIEW

**Review by alecghica — APPROVED** [review]

File: src/pas/plugins/authomatic/plugin.py [hunk 1]
```diff
@@ -92,6 +92,10 @@ def lookup_identities(self, result):
     @security.private
     def remember_identity(self, result, userid=None):
         """stores authomatic result data"""
+        if not getattr(result.user, "id", None):
+            # fetch provider specific user-data
+            result.user.update()
+
         if userid is None:
```
[the remark this answers is not in the document]
Comment (avoinea) [reply src/pas/plugins/authomatic/plugin.py:99]: Without this, you'll not be able to login on frontend if already logged-in on backend (Plone Classic). Backend will fail with:
```
AttributeError: user.id
```

[the remark this answers is not in the document]
Comment (jensens) [reply src/pas/plugins/authomatic/plugin.py:99]: maybe @ericof knows what happens at the restapi level here. 
The idea is to store identities once for each user, so this is a bug AFAICT.

[the remark this answers is not in the document]
Comment (avoinea) [reply src/pas/plugins/authomatic/plugin.py:99]: @jensens @ericof Hmm, not good :thinking: 

Screenshot from 2024-06-17 11-05-52

[the remark this answers is not in the document]
Comment (avoinea) [reply src/pas/plugins/authomatic/plugin.py]: Removed this fix. I think the current approach is more safe.

**Comment by jensens** [comment]
LGTM, but conflicts need resolution before merge

**Comment by avoinea** [comment]
@jensens Done 😉

**Pull request merged**